Justfile

```diff
@@ -0,0 +1,24 @@
+_default:
+    @just --choose
+
+# Shows a list of all available recipes
+help:
+    @just --list
+
+green := '\033[0;32m'
+red := '\033[0;31m'
+reset := '\033[0m'
+
+# Checks if all requirements to work on this project are installed
+check-requiements:
+    @command -v cargo &>/dev/null && echo -e "{{ green }}✓{{ reset}} cargo installed" || echo -e "{{ red }}✖{{ reset }} cargo missing"
+
+# Runs the same linters as the pipeline with fix option
+lint:
```

Comment:
Can we use these from the CI, instead of having dublicate code there?

Reply:
well of course you could use just or make in the ci, but not the other way arroud, unless you manuallly parse the yml of the workflow, and then you allways would need to update the parser on ci config updates

Reply:
also here we use different options on purpose, because on local machines we want to run it with the fix optins (or without the check option)

Reply:
so yea, not really suggested to try parsing yml as shell commands.
I will resolve this one, because  that really shouldn't be done, especially, because it isn't even full duplicated line because of the options. the minor-minor duplication there is more than acceptable

Reply:
> well of course you could use just or make in the ci, but not the other way arroud

I meant the former, but well...